config/routes.rb

```diff
@@ -27,6 +27,11 @@
   get "/sign-in/callback", to: "sessions#callback", as: :new_govuk_session_callback
   get "/sign-out", to: "sessions#delete", as: :end_govuk_session
 
+  scope "/account" do
+    get "/save-a-page/new", to: "save_pages#create", as: :save_page
+    get "/save-a-page/remove", to: "save_pages#destroy", as: :remove_saved_page
```

Comment:
I think it should be `/saved-pages`, then we have

- `/account` - the root
- `/account/saved-pages` - the feature
- `/account/saved-pages/new` - the action